### PR TITLE
DOCS-4386

### PR DESCRIPTION
**Pull request by valkyrie69**
Doc refresh. Update for how to add a server to a cluster. Customer feedback--instructions were not clear. There have been some udpates to the UI.

### CODE REVIEW

**Review by mcocaro — APPROVED** [review]

File: modules/ROOT/pages/cluster-add-srv.adoc [hunk 1]
```diff
@@ -3,21 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To add a server to a cluster, all servers in a cluster must run the same version of Mule runtime and of the Runtime Manager Agent.
+To add a server to a cluster, all servers in a cluster must run the same version of Mule runtime and of the Runtime Manager agent.
 
 [NOTE]
-Remember that you must first register each of the servers in the cluster by downloading and installing the same version of the https://www.mulesoft.com/platform/mule[Mule] and then running the xref:managing-servers.adoc#add-a-server[amc_setup] script on each.
-A cluster can only be made up of servers that are not already in a server group or part of another cluster.
+You must first register each of the servers in the cluster by downloading and installing the same version of the https://www.mulesoft.com/platform/mule[Mule] and then running the xref:managing-servers.adoc#add-a-server[amc_setup] script on each.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/cluster-add-srv.adoc]: Register the servers before what? I'd delete "You must" and just make this a step. I don't understand why this is a note when it is a requirement for something. This is a step, not a note...?

File: modules/ROOT/pages/cluster-add-srv.adoc [hunk 2]
```diff
@@ -3,21 +3,20 @@ ifndef::env-site,env-github[]
...
+. In the server list, under *Type*, click the name of the cluster to which to add the server. +
+The management panel opens on the right side of the page.
 +
 image::server-details.png[]
 
-. Click *Add Servers*.
-+
-image::add-servers-button.png[AddServers]
+. Click *Add Server*. +
+You can add only servers that are not already in a server group or part of another cluster.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/cluster-add-srv.adoc]: A little awkward? (just me?) Are servers already in a group/part grayed out? What prevents "me" from selecting them. I'd write it "Add  the servers you wish to include in the group. Servers that belong to another group or cluster are grayed out (or whatever indicates you can't choose them).  It's longer but the two concepts are broken out into two separate sentences ...

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/cluster-add-srv.adoc]: Oh, and then you don't need most of the sentence in line 20

File: modules/ROOT/pages/cluster-create.adoc [hunk 3]
```diff
@@ -5,53 +5,49 @@ endif::[]
...
 
-Also, note the following considerations when creating a cluster:
-
-* None of the servers you select for creating a new cluster can contain any *previously deployed applications*.
-
-* Multicast servers can be in the status states of *Running* or *Disconnected*, whereas unicast servers can be only in the *Running* state. 
-
+[NOTE]
+--
+* None of the servers you select for a new cluster can contain any previously deployed applications.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/cluster-create.adoc]: nor can they be members of another cluster/group. That'd be good to include here?

File: modules/ROOT/pages/cluster-create.adoc [hunk 4]
```diff
@@ -5,53 +5,49 @@ endif::[]
...
+* Multicast servers can have status states of *Running* or *Disconnected*, whereas unicast servers must be in the *Running* state. 
 * All servers in a cluster must be running the same Mule runtime engine version and Runtime Manager agent version.
+* To have Object Store persistence on all the servers in your cluster, you must create your cluster manually and use Quorum Management.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/cluster-create.adoc:19]: Do I know what multicast, unicast servers are ? Can we link to doc about them in the See Also?

File: modules/ROOT/pages/cluster-create.adoc [hunk 5]
```diff
@@ -5,53 +5,49 @@ endif::[]
...
+* Be unique
+* Contain only letters, numbers, or dashes
+.. Select *Unicast* or *Multicast*.
+.. In the list of serves, select the servers to add to your cluster, then click *Create Cluster*. +
+The cluster appears in the server list. 
+. Click the cluster name to view details about the cluster. In the cluster details page, click the *Actions* drop-down to: +
+* Delete servers from the cluster
+* Restart servers in the cluster
+* Shut down servers in the cluster
+* Renew the certificate for servers in the cluster.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/cluster-create.adoc]: either lines 44-47 should all have a period at the end or none of them. Are these also the UI labels for the actions? Should we make them have UI label (bold) font?

**Review by Mulesoft-Mysti — APPROVED** [review]
just a few things I noticed. None are showstoppers.

**Pull request merged**